client/src/connection/index.ts

```diff
@@ -19,6 +19,7 @@ export interface Session {
   setup(): Promise<void>;
   run(code: string, onLog?: (logs: LogLine[]) => void): Promise<RunResult>;
   close(): Promise<void> | void;
+  sessionId?(): string | undefined;
```

Comment:
Trying to remember on this one. Did we promote the sessionId up so that it would make caching/reuse easier based on the discussions we had last week? @JosephDHenry

Reply:
Joseph's changes are here: https://github.com/sassoftware/vscode-sas-extension/pull/175 . My best guess is we'll have to circle back and make adjustments after his changes are merged